### ISSUE TITLE
Tidy up Repo 

### ISSUE THREAD

**Issue by GeorgiaHCA** [alan-turing-institute/AutisticaCitizenScience#415]
## Summary

Make sure the repo is in really good condition for further engagement/community building with project. 

## Date Due by

This is an ongoing process, but I'm going to aim to finish this iteration of tidying work by **Thursday 18th February**

## Details

It's been a while since the last refresh and time for an early Spring clean!

## Suggestions for tasks that can fix this issue

* [ ] Make links are working - look out for 404s! 
* [ ] All information up-to-date,
* [ ] All data up-to-date - including Good form survey
* [ ] Steps to involvement for different groups clear
* [ ] Get repo tested with different members of the community 
* [ ] Check for typos, poor style, etc.
* [ ] Update labels and milestones 

* [ ] (additional) - add anything new that would help with engagemeent or access. 

## Who can help

Anyone can help! Feel free to create PRs to fix any problems you find, or just leave a comment or make an issue for any problems you encounter you aren't sure how to fix. 
Autistic people especially encouraged to help 
You can also just comment if anything is confusing or difficult for you to access

**Comment by GeorgiaHCA** [comment]
Let's try to get tihs done by early April - and iterate from there!

**Comment by GeorgiaHCA** [comment]
I did a partial Spring clean before I went on leave on April, but I still think this repository needs some more updating and cleaning as well as deep thinking. 

One of my priority tasks is to continue to update the repository with information and make sure it's findable.

**Comment by KirstieJane** [comment]
Is this the right task for you to work on @GeorgiaHCA ? Can @lottycoupat / @sowmya9507 take the lead or is there too much that's in your head for them to take the work forwards?

**Comment by GeorgiaHCA** [comment]
Happy for anyone to take the lead @KirstieJane. I think the only thing I will need to do myself is add anything which I haven't added yet.

**Comment by lottycoupat** [comment]
@GeorgiaHCA @KirstieJane @sowmya9507 I'm happy to take the lead on this one, already something I've been thinking about. The checklist there definitely gives a direction also so super useful!

**Comment by GeorgiaHCA** [comment]
Fantastic @lottycoupat you are a 🌟

**Comment by sowmya9507** [comment]
@GeorgiaHCA I am so sorry I couldn't reply on Friday, since I had intense classes going on. I can definitely help out in tidying up the repository and checking what links are working and what not. If required, we can also make the requirement lists more of a markdown, since I think it is pretty useful for the newcomers in the project :) Let me know your thoughts on this. 😄

**Comment by GeorgiaHCA** [comment]
Heya @sowmya9507 no worries at all - hope your classes were interesting! If you have time to help with tidying the repository, that would be briliant! 🌟 

There is a markdown of the requirements list in the Autspaces repo: https://github.com/alan-turing-institute/AutSPACEs/blob/main/platform-design/requirements-list.md - but I don't think it's very findable and is mingled with other suggestions. Do you think it should be in this repository too? I also didn't include the sources, although I have a longer copy which has them included - or would this be confuding to developers? Open to suggestions!

**Comment by sowmya9507** [comment]
@GeorgiaHCA thank you 😄 
Yes. I viewed the markdown file, but personally for me when I started exploring the repository I did not immediately find the list. Maybe a link or findable option to go to this link might be helpful? And in the case of developers, I guess the more info provided the better. At least for newbies like me 😃

**Comment by lottycoupat** [comment]
@KirstieJane @GeorgiaHCA @sowmya9507 

Thoughts on merging and having one core repository for the whole project? Following the Turing Way repo which includes development and coding of the website but also communication/meeting notes? Rather than what we have at the moment which is split over two repos... 

Would love to get your thoughts and views on this! 

Suggestions:

1) Merge Repos (AutSPACEs repo as **core repo**) 

- Tidy up platform development folders to be more intuitive 
- Add folders on communications and meeting to include notes
- Keep this repo as an audit of past work from Fujitsu and other documentation

2) Keep repos separate

- Tidy each repository to make more intuitive
- Link to meeting notes and information from this repo in the AutSPACEs repo 
- AutSPACEs repo specifically platform development __only__

**Comment by GeorgiaHCA** [comment]
Heya @lottycoupat I think this is a great idea and they should def be merged into one place, especially since the dev tasks are so connected/not separable from the platform management tasks

**Comment by lottycoupat** [comment]
UPDATE:

Would love your thoughts on this all @GeorgiaHCA @KirstieJane @sowmya9507 

In terms of repo refresh, I have spent the day tidying some things up. This has included considerate time spent on learning basic github skills such as renaming 😆 

I have created a few pull requests for review:
- #486 Adding notes from meetings - also creates a new folder in project-management called project-meetings
- #484 added user-flow folder including readme and image of the user flow to the platform-designs folder
- #487 moving requirements from AutSPACEs repo to Autistica repo - all documentation like this should be transferred to the Autistica repo [PR 109](https://github.com/alan-turing-institute/AutSPACEs/pull/109)
- #488 adding a platform-code folder that includes simply a readme directing users to the AutSPACEs repo
- #490 moving GSoC documents to the Autistica repo - then needs to be deleted from AutSPACEs repo [PR 108](https://github.com/alan-turing-institute/AutSPACEs/pull/108)

Sowmya successfully merged adding wireframe photos to the Autistica repo 🎉 #483
